### PR TITLE
"Cloud Response - generic" Fixes

### PR DESCRIPTION
**Pull request by ssokolovich**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-6553)

## Description
Added fixes to the generic playbook and the sub-playbooks:
- Choose manually the cloud provider
- Removed all the alertJson old values
- Changed data collection titles

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_17.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Cloud Response - AWS
+
+- Fixed Data Collection titles.
+- Fixed tasks using `alertJson` context path.
```
Comment (melamedbn) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_17.md:7]: Didn't see any removal of 'alertJson'

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ description: |-
   - Terminate/Shut down/Power off an instance
   - Delete/Disable a user
   - Delete/Revoke/Disable credentials
-  - Block indicators
+  - Block indicators.
```
Comment (melamedbn) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml:14]: Let's remove the period

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_17.md (hunk 1 above)
Comment (ssokolovich) [reply Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_17.md:7]: indeed, great catch!

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml [hunk 3]
```diff
@@ -1785,19 +1745,19 @@ tasks:
...
       condition:
       - - operator: isEqualString
           left:
             value:
-              simple: identity_invoked_by_type
+              simple: inputs.accountType
             iscontext: true
           right:
             value:
-              simple: service_account
```
Comment (melamedbn) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml:1800]: Why did you change it?

**Review by melamedbn — APPROVED** [review]

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml (hunk 3 above)
Comment (ssokolovich) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml:1800]: I thought it would be easier to use it without `_`, 'cause it will be controlled through the input. Do you see an issue using that in it's newer version?
